app/views/recipes/_form.html.erb

```diff
@@ -85,22 +85,18 @@
       </div>
     </div> <!-- row -->
 
-    <table class="table table-ingredients">
-      <thead>
-        <tr>
-          <th>Qty</th>
-          <th>Measurement</th>
-          <th>Name</th>
-          <th>Preparation Style</th>
-          <th>Delete?</th>
-        </tr>
-      </thead>
-      <tbody>
-        <%= form.fields_for :ingredients do |f_ingredient| %>
-          <tr><%= render 'ingredients/form_fields', f_ingredient: f_ingredient %></tr>
-        <% end %>
-      </tbody>
-    </table>
+    <section id='ingredients-container'>
+      <header id='ingredients-header'>
+        <div class="ingredients-col flex-sm">Qty</div>
+        <div class="ingredients-col">Measurement</div>
+        <div class="ingredients-col">Name</div>
+        <div class="ingredients-col">Prep Style</div>
+        <div class="ingredients-col flex-xs"><%= MaterialIcon.new(icon: :trash, size: :medium).render %></div>
+      </header>
```

Comment:
I don't know how you prefer to name and organize styles and stuff. I just was trying to get it to work